wicket-core/src/main/java/org/apache/wicket/protocol/http/CsrfPreventionRequestCycleListener.java

```diff
@@ -178,6 +181,8 @@ public String toString()
 	 */
 	private Collection<String> acceptedOrigins = new ArrayList<>();
 
+	private final ResourceIsolationPolicy fetchMetadataPolicy = new DefaultResourceIsolationPolicy();
```

Comment:
I think this policy should be replaceable/configurable here, i.e. should be passed as constructor parameter.
The default constructor should use `DefaultResourceIsolationPolicy`